pom.xml

```diff
@@ -782,6 +784,26 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <plugin>
+                <groupId>io.github.git-commit-id</groupId>
+                <artifactId>git-commit-id-maven-plugin</artifactId>
```

Comment:
Unrelated to this change, but in the `kroxylicious-app/src/main/templates/metadata.properties`, which gets interpolated with some of the properties this thing defines, why did we add `git.commit.message.short=${git.commit.message.short}` for use in the --version message. That seems a bit weird to me.